perprof/matplotlib.py

```diff
@@ -92,9 +92,20 @@ def plot(self):
         else:
             linestyles = ['b', 'g', 'r', 'c', 'm', 'y']
 
+        times = [self.times[0]]
```

Comment:
Instead of going from (a,b) to (c,d), I have to go (a,b) -> (c,b) -> (c,d), so the x's (times) are doubled, except the first, and the y's (ppsbt) are doubled, except the last.